modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/RuleList/RuleList.es.js

```diff
@@ -158,6 +191,34 @@ const configDefault = {
 	spritemap,
 };
 
+const configNumeric = {
+	pages: pagesWithNumbers,
```

Comment:
@alinedoleron, can you please declare the `pagesWithNumbers` just before the `configNumeric`?

From what I could see from this code, the variables are being declared using the alphabetic order except for the variables that are dependencies of other variables, which is the case of variables `pages` and `pagesWithNumbers`.